service/src/main/java/uk/gov/hmcts/reform/fpl/service/RespondentService.java

```diff
@@ -84,4 +89,32 @@ public AboutToStartOrSubmitCallbackResponse addHiddenValues(CaseDetails caseDeta
             .data(data)
             .build();
     }
+
+    public String buildRespondentLabel(List<Element<Respondent>> respondents) {
+        StringBuilder sb = new StringBuilder();
+
+        if (isNotEmpty(respondents)) {
+            AtomicInteger i = new AtomicInteger(1);
+
+            respondents.forEach(respondent -> {
+                sb.append("Respondent")
+                    .append(" ")
+                    .append(i)
+                    .append(" ")
+                    .append("-")
+                    .append(" ")
+                    .append(defaultIfNull(respondent.getValue().getParty().firstName, ""))
```

Comment:
I saw piece of logic in one of the other PRs that added `getFullName` (or similar name) to the party class that does concatenate first name and last name. It could be reused here when available.